gateway/webhook/webhookTransformer.go

```diff
@@ -74,16 +82,17 @@ func (bt *batchWebhookTransformerT) transform(events [][]byte, sourceType string
 							userId: "U123"
 						}
 					]
+				},
+				outputToSource: {
```

Comment:
instead of having a single example response with all fields filled, which according to my understanding is not a valid scenario, wouldn't it make sense to have 3 example responses, 1 for each main scenario?